lektor/imagetools.py

```diff
@@ -331,7 +332,8 @@ def get_image_info(fp):
     if any(map(head.strip().startswith, magic_bytes)):
         return get_svg_info(fp)
 
-    fmt = imghdr.what(None, head)
+    _type = filetype.image(head)
```

Comment:
This will throw if the format is not recognized. Looking at the `imghdr.what` source code it will never throw. Should we catch the exception here?

Reply:
It doesn't throw, it returns None (see the next line). The exception is only in py2 due to the thing below.

Reply:
You are correct I think. I got confused by the documentation: https://h2non.github.io/filetype.py/v0.1.3/match.m.html#filetype.match.image

Reply:
Yeah, the docs are pretty lacking.